components/org.wso2.carbon.identity.oidc.session/src/main/java/org/wso2/carbon/identity/oidc/session/servlet/OIDCLogoutServlet.java

```diff
@@ -182,32 +183,42 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response)
                 handleLogoutResponseFromFramework(request, response);
                 return;
             }
+            String idTokenHint = request.getParameter(OIDCSessionConstants.OIDC_ID_TOKEN_HINT_PARAM);
             // Get user consent to logout
-            boolean skipConsent = getOpenIDConnectSkipeUserConsent();
-            if (skipConsent) {
-                String idTokenHint = request.getParameter(OIDCSessionConstants.OIDC_ID_TOKEN_HINT_PARAM);
-                if (StringUtils.isNotBlank(idTokenHint)) {
-                    redirectURL = processLogoutRequest(request, response);
-                    if (StringUtils.isNotBlank(redirectURL)) {
-                        response.sendRedirect(getRedirectURL(redirectURL, request));
-                        return;
+            try {
+                boolean skipConsent = getOpenIDConnectSkipUserConsent(idTokenHint);
```

Comment:
We can limit the try block to this line only. Are other logics also changed with this